assets/js/base/context/providers/add-to-cart-form/form-state/index.js

```diff
@@ -245,7 +245,7 @@ export const AddToCartFormStateContextProvider = ( {
 						const message =
 							data.processingResponse?.message ||
 							__(
-								'Something went wrong. Please contact us to get assistance.',
+								'Something went wrong. Please contact us for assistance. Please contact us to get assistance.',
```

Comment:
This seems duplicated. The same is happening in four more of the `Please contact us to get assistance.` messages.

Reply:
Wow thanks a lot for spotting that @albarin !